h/schemas/admin_group.py

```diff
@@ -0,0 +1,66 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import unicode_literals
+
+import colander
+from deform.widget import SelectWidget, TextInputWidget
+
+from h import i18n
+from h import validators
+from h.models.group import (
+    GROUP_NAME_MIN_LENGTH,
+    GROUP_NAME_MAX_LENGTH,
+    GROUP_DESCRIPTION_MAX_LENGTH
+)
+from h.schemas.base import CSRFSchema
+
+_ = i18n.TranslationString
+
+VALID_GROUP_TYPES = (
```

Comment:
Talked to @seanh some about this today—we both agreed that it would be ideal sometime soonish to have the group service or similar have a method for returning the valid types of groups and their associated create functions. Right now, I'm hard-coding this to get things rolling, as there is no such extant source of "what types of groups" are available.